sirepo/package_data/static/js/sirepo-plotting.js

```diff
@@ -5,7 +5,7 @@ var srdbg = SIREPO.srdbg;
 SIREPO.PLOTTING_LINE_CSV_EVENT = 'plottingLineoutCSV';
 SIREPO.DEFAULT_COLOR_MAP = 'viridis';
 
-SIREPO.app.factory('plotting', function(appState, d3Service, frameCache, panelState, utilities, requestQueue, simulationQueue, $interval, $rootScope, $window) {
+SIREPO.app.factory('plotting', function(appState, frameCache, panelState, utilities, requestQueue, simulationQueue, $interval, $rootScope, $window) {
 
     var INITIAL_HEIGHT = 400;
     var MAX_PLOTS = 11;
```

Comment:
11? I'm sure there's a reason for this, but why?

Reply:
MAX_PLOTS is actually the max number of lines to show on a convergence plot. The constant should be renamed to MAX_CONVERGENCE_LINES or something similar.